### ISSUE TITLE
No more unlocked exercises

### ISSUE THREAD

**Issue by yanakad** [exercism/exercism#5215]
Hi all, 
I started the track, completed the first 2 exercises and some additional/optional ones that got unlocked. At this point, I don't see anything else that could be completed. The bulk of the exercises is still locked for me.
What do I need to do to unlock more exercises? I cannot see any instructions on "Next step"/where to go from here

I see an issue in the Ruby track https://github.com/exercism/exercism/issues/4096 but I'm doing Rust

**Comment by iHiD** [comment]
Hello. Are you in Mentored Mode and if so, are you waiting for a mentor to review a solution before you can continue?

**Comment by yanakad** [comment]
thanks -- you were, in fact, correct. My last exercise was in "approved" state. I didn't realize I need to transition it to "Completed" before I can proceed. After your comment I tried and now I can see the next stage unlocked. 
FWIW would be nice if "Approved" also unlocked or if we could somehow have an indicator of a "Needs user action" vs "Done" state. "Approved" very much sounded like "Done" to me

Thanks for your help, my issue is now resolved.
